src/gg.regression.unity.experimental.runespawn/Editor/Scripts/PlayModeController.cs

```diff
@@ -68,6 +72,25 @@ public static void StartPlayMode(TcpClient client)
             EditorApplication.delayCall += () =>
             {
                 Application.runInBackground = true;
+
+                _botSequencePath = botSequencePath;
```

Comment:
I dislike the usage of global variables here. 

I also tried using a lambda to make a closure (is this the correct term?) / bake the botSequencePath into the delegate, however it made unsubscribing finnicky.   

```csharp
                EditorApplication.playModeStateChanged += (PlayModeStateChange state) =>
                {
                    RunBotSequenceOnPlayModeEnter(state, botSequencePath);
                };
```

Thoughts?

Reply:
I think it's fine in this case, but make sure to reset the variable when the sequence is done, has failed, or when a new change request has come in maybe? I'm not too sure about our static variable usage since we usually go with an instanced singleton class, but also since you already have static _logs and _errors, I don't see this being any different from those.